kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/channel/Coordinator.java

```diff
@@ -296,19 +302,18 @@ private Map<Integer, Long> lastCommittedOffsetsForTable(Table table, String bran
     return ImmutableMap.of();
   }
 
-  @Override
-  void stop() {
```

Comment:
[Q] wondering if we should still call Channel#stop()

Reply:
We still call it to clean up the Kafka consumer and producer. The consumer isn't thread safe so I left that call on the Coordinator thread, rather than the main task thread. It isn't ideal having both a `terminate` and a `stop`, though this was the best I came up with.